src/dashboard/src/components/accounts/views.py

```diff
@@ -164,3 +193,8 @@ def delete(request, id):
         return redirect('components.accounts.views.list')
     except:
         raise Http404
+
+
+def logged_out(request):
```

Comment:
This is a simple 'logged out' page and is intended to be login-exempt, so there is somewhere to send the user post-logout that won't bounce them straight back to a login form or being logged straight back in again (in the case we don't do full IdP logout). This will depend on exactly how the logout flow works, which is still uncertain.